packages/app-desktop/gui/ConfigScreen/controls/plugins/PluginBox.tsx

```diff
@@ -217,6 +220,17 @@ export default function(props: Props) {
 		/>;
 	}
 
+	const renderDefaultPluginLabel = () => {
+		// Built-in plugins can only be disabled
+		if (item.manifest._built_in) {
+			return (
+				<BoxedLabel>{_('Built in')}</BoxedLabel>
```

Comment:
I wonder if that would scale well in other languages since the the plugin box has a fixed width? If it is an issue we could maybe replace it with an icon and a tooltip

Reply:
A screenshot of the current label:
![screenshot: Text "Built in" surrounded by a grey border in the bottom right of the plugin card](https://github.com/laurent22/joplin/assets/46334387/cfb6f9b3-1cdc-41c7-b537-2e78f166238d)

With a long label:
![screenshot: Text "Labels can be long. A very long label..." still doesn't fill the width of the card.](https://github.com/laurent22/joplin/assets/46334387/dc875d25-24cd-4966-904e-aa068c96d3e3)